packages/type-utils/package.json

```diff
@@ -58,6 +58,7 @@
   "devDependencies": {
     "@jest/types": "29.6.3",
     "@typescript-eslint/parser": "8.18.0",
+    "@typescript/vfs": "*",
```

Comment:
y?

```suggestion
```

Reply:
🤷‍♂️ 

https://github.com/typescript-eslint/typescript-eslint/blob/bc12e04be86f7c7dd42d4f2a02f1cf51f35bf067/packages/type-utils/tests/getConstrainedTypeAtLocation.test.ts#L46-L76

Reply:
❤️

Reply:
Actually, I may be able to copy testing patterns from within this repo, and avoid that, lemme poke around.

Reply:
Yeah, removed all that vfs stuff. Turns out, we already infra for parsing and using type information. _Who would have guessed?_ 🤦‍♂️